tests/test_app.py

```diff
@@ -180,3 +184,17 @@ def test_article_feed_adds_article(aa_data, articles_data):
 def test_articles_skips_articles_without_required_fields():
     arts = list(articles())
     assert len(arts) == 1
+
+
+@freeze_time("2023-08-18")
```

Comment:
Is this necessary if we're not actually checking the date in this unit test?

Reply:
This is required by the stubbed SNS client, which expects the date to be included in the `Message` parameter when calling `sns_client.publish()`.

Reply:
Got it, thanks for the clarification!